addons/project_issue/models/project_issue.py

```diff
@@ -0,0 +1,303 @@
+    email_cc = fields.Char('Watchers Emails', help="""These email addresses will be added to the CC field of all inbound
+        and outbound emails for this record before being sent. Separate multiple email addresses with a comma""")
+    date_open = fields.Datetime('Assigned', readonly=True, index=True)
+    # Project Issue fields
+    date_closed = fields.Datetime('Closed', readonly=True, index=True)
+    date = fields.Datetime()
+    date_last_stage_update = fields.Datetime('Last Stage Update', index=True, default=lambda self: datetime.now())
+    channel = fields.Char(help="Communication channel.")
+    tag_ids = fields.Many2many('project.tags', string='Tags')
+    priority = fields.Selection([('0', 'Low'), ('1', 'Normal'), ('2', 'High')], 'Priority', index=True, default='0')
+    stage_id = fields.Many2one('project.task.type', 'Stage', track_visibility='onchange', index=True,
+                               domain="[('project_ids', '=', project_id)]", copy=False,
+                               default=lambda self: self.stage_find(self.env.context.get('default_project_id'), [('fold', '=', False)]))
+    project_id = fields.Many2one('project.project', 'Project', track_visibility='onchange', index=True)
+    duration = fields.Float()
+    task_id = fields.Many2one('project.task', 'Task', domain="[('project_id','=',project_id)]",
+                              help="You can link this issue to an existing task or directly create a new one from here")
+    day_open = fields.Float(compute='_compute_day', string='Days to Assign', store=True)
+    day_close = fields.Float(compute='_compute_day', string='Days to Close', store=True)
+
+    user_id = fields.Many2one('res.users', 'Assigned to', index=True, track_visibility='onchange', default=lambda self: self.env.uid)
+    working_hours_open = fields.Float(compute='_compute_day', string='Working Hours to assign the Issue', store=True)
+    working_hours_close = fields.Float(compute='_compute_day', string='Working Hours to close the Issue', store=True)
+    inactivity_days = fields.Integer(compute='_compute_days_since_creation_inactivity_days', string='Days since last action',
+                                     help="Difference in days between last action and current date")
+    color = fields.Integer('Color Index')
+    user_email = fields.Char(related='user_id.email', string='User Email', readonly=True)
+    date_action_last = fields.Datetime('Last Action', readonly=True)
+    date_action_next = fields.Datetime('Next Action', readonly=True)
+
+    _group_by_full = {
+        'stage_id': _read_group_stage_ids
+    }
+
+    @api.depends('date_open', 'date_closed', 'working_hours_close', 'working_hours_close', 'day_open', 'day_close')
+    def _compute_day(self):
+        Calendar = self.env['resource.calendar']
+        for issue in self.filtered('create_date'):
+            calendar_id = None
+            if issue.project_id and issue.project_id.resource_calendar_id:
```

Comment:
remove condition as resource_calendar_id is recordset.if no value it's return empty recordset.